src/wallet/db.cpp

```diff
@@ -692,7 +692,7 @@ void BerkeleyEnvironment::Flush(bool fShutdown)
 {
     int64_t nStart = GetTimeMillis();
     // Flush log data to the actual data file on all files that are not in use
-    LogPrint(BCLog::DB, "BerkeleyEnvironment::Flush: Flush(%s)%s\n", fShutdown ? "true" : "false", fDbEnvInit ? "" : " database not started");
+    LogPrint(BCLog::DB, "BerkeleyEnvironment::Flush: [%s] Flush(%s)%s\n", strPath, fShutdown ? "true" : "false", fDbEnvInit ? "" : " database not started");
```

Comment:
Perhaps change order to that the static message comes before the situation specific message in order to increase readability?

That is change from  …

```
BerkeleyEnvironment::Flush: [/Users/joao/Library/Application Support/Bitcoin/regtest/wallets/w1] Flush(true)
```

… to say …

```
BerkeleyEnvironment::Flush: Flush(true), file: /Users/joao/Library/Application Support/Bitcoin/regtest/wallets/w1
```

Reply:
nah, context before message, message before context, you could make a valid point to go either way